src/main/java/edu/mum/sonet/models/BaseEntity.java

```diff
@@ -0,0 +1,20 @@
+package edu.mum.sonet.models;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@MappedSuperclass
+public class BaseEntity {
```

Comment:
BaseEntity needs to implement Serializable

Reply:
And also specify GenerationType.IDENTITY because we don't have a configuration file for that.

Reply:
i did it but it choose the default GenerationType.AUTO